front/app/containers/Admin/projects/projectFolders/components/ProjectFolderRow/FolderMoreActionsMenu.tsx

```diff
@@ -35,11 +40,12 @@ const FolderMoreActionsMenu = ({ folderId, setError }: Props) => {
     handler: async () => {
       if (window.confirm(formatMessage(messages.deleteFolderConfirmation))) {
         setIsDeleting(true);
+        setIsRunningAction && setIsRunningAction(true);
```

Comment:
Could remove the checks once we make setIsRunningAction required. :-)

Reply:
Resolving this since conversation similar below